### DISCUSSION TITLE
Ability to watch stores in Alpine 3

### DISCUSSION THREAD

**Discussion by galengidman** [alpinejs/alpine#1737] [1. Help] [answered]
With Alpine 2 + and @ryangjchandler's Spruce, there was the ability to watch stores with `Spruce.watch` in js, or via `$watch('$store.foo')` magics. I'm not seeing the same functionality in Alpine 3's stores features — am I missing something?

**Comment by danddanddand** [comment, marked as answer]
There are a few ways discussed here https://github.com/alpinejs/alpine/discussions/1653

**Reply by galengidman** [reply]
Thanks for pointing me in the right direction.
